Microsoft.Toolkit.Uwp.UI.Controls.Ribbon/RibbonSplitButton.cs

```diff
@@ -1,56 +1,40 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Runtime.InteropServices.WindowsRuntime;
-using Windows.UI.Xaml;
+﻿using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Data;
-using Windows.UI.Xaml.Documents;
-using Windows.UI.Xaml.Input;
-using Windows.UI.Xaml.Media;
-
-// The Templated Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234235
 
 namespace Microsoft.Toolkit.Uwp.UI.Controls.Ribbon
 {
-    public sealed class RibbonSplitButton : Control
+    public sealed class RibbonSplitButton : Button
```

Comment:
This is the only major takeaway. It fixes the issue with the SplitButton not working.

Reply:
Why inherit from `Button` and not `SplitButton`?

Reply:
SplitButton should be fine as long as it has Command and CommandParameter properties (which I'm pretty sure it does).

 It was previously set to `Control` which did not have any of the properties that you were using in the ControlTemplate. (PrimaryButton had ~6 broken TemplateBindings).

The hierarchy looks like this Control >  ContentControl > SplitButton. see https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.splitbutton?view=winrt-19041